### ISSUE TITLE
sfdx force:org:list returns exit code 1

### ISSUE THREAD

**Issue by fenixjke** [forcedotcom/cli#1322]
**Is your feature request related to a problem? Please describe.**
In the systems that don't have an authorized org, the "force:org:list" command returns the error instead of empty result. Especially in CI/CD processes we need to create additional conditions to handle that specific situation.

**What are you trying to do**
`sfdx force:org:list --json` returns status and exist code 1 which indicate that the command has been executed unsuccessfully, but the empty result is expected.
The outcome is
```
{
	"status": 1,
	"name": "noOrgsFound",
	"message": "No orgs can be found",
	"exitCode": 1,
	"actions": [
		"Use one of the auth commands or force:org:create to add or create a scratch org."
	],
	"commandName": "OrgListCommand",
	"stack": "...",
	"warnings": []
}
```

**Describe the solution you'd like**
Not having authorized orgs shouldn't lead to errors when performing the org:list commands. Understanding that currerntly there is no authorized orgs should should return status 0 and show the result like this:
```
{
	"status": 0,
	"result": {
		"nonScratchOrgs": [],
		"scratchOrgs": []
	}
}
```

**Describe alternatives you've considered**
Currently, this situation is considered as incorrect by the library and requires additional handlers in "exception" section

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10308950](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Xo0OXYAZ/view)

**Comment by WillieRuemmele** [comment]
Hey @fenixjke I'm just getting through some of our backlog, and just confirmed this is working as you suggested
![image](https://user-images.githubusercontent.com/23369140/179602734-3c083dfd-6b3b-41c0-ae33-502cc7394573.png)
on 
```
{
	"cliVersion": "sfdx-cli/7.159.0",
	"architecture": "darwin-x64",
	"nodeVersion": "node-v16.14.2",
	"pluginVersions": [
		"@oclif/plugin-autocomplete 0.3.0 (core)",
		"@oclif/plugin-commands 1.3.0 (core)",
		"@oclif/plugin-help 3.3.1 (core)",
		"@oclif/plugin-not-found 1.2.6 (core)",
		"@oclif/plugin-plugins 1.10.11 (core)",
		"@oclif/plugin-update 1.5.0 (core)",
		"@oclif/plugin-warn-if-update-available 1.7.3 (core)",
		"@oclif/plugin-which 1.0.4 (core)",
		"@salesforce/lwc-dev-server 2.11.0",
		"@salesforce/sfdx-plugin-lwc-test 1.0.0 (core)",
		"alias 2.1.0 (core)",
		"apex 1.1.0 (core)",
		"auth 2.2.2 (core)",
		"community 2.0.0 (core)",
		"config 1.4.14 (core)",
		"custom-metadata 2.0.0 (core)",
		"data 2.0.4 (core)",
		"generator 2.0.2 (core)",
		"info 2.0.1 (core)",
		"limits 2.0.1 (core)",
		"org 2.0.2 (core)",
		"release-management 2.13.0",
		"└─ trust 1.1.0",
		"salesforce-alm 54.6.2 (core)",
		"schema 2.1.1 (core)",
		"sfdx-cli 7.159.0 (core)",
		"signups 1.2.0 (core)",
		"source 2.0.7 (core)",
		"telemetry 2.0.0 (core)",
		"templates 55.0.0 (core)",
		"user 2.1.0 (core)"
	],
	"osVersion": "Darwin 21.5.0"
}
```

I'll close this, but please reopen if you're seeing different results
